app/mailers/adoption_application_mailer.rb

```diff
@@ -0,0 +1,12 @@
+class AdoptionApplicationMailer < ApplicationMailer
+  def new_adoption_application(adoption_application)
+    @adoption_application = adoption_application
+    mail(to: admin_email, subject: "A new adoption application has been submitted")
+  end
+
+  private
+
+  def admin_email
+    ENV['ADMIN_EMAIL']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.